crypto-keys/src/main/java/com/palantir/crypto2/keys/serialization/KeyMaterials.java

```diff
@@ -96,22 +95,22 @@ public static byte[] symmetricWrap(KeyMaterial keyMaterial, SecretKey key) {
 
     public static KeyMaterial unwrap(byte[] wrappedKeyMaterial, PrivateKey key) {
         int version = version(wrappedKeyMaterial);
-        checkArgument(
+        Preconditions.checkArgument(
```

Comment:
nit: mind statically importing this?

Reply:
Sure, changed this usage so it kept the same format as before my change. I'll note that most of the other files didn't use static imports for Preconditions (even before my change) though